### ISSUE TITLE
There are a lot of things that aren't plugins

### ISSUE THREAD

**Issue by YaBoiBurner** [vim-awesome/vim-awesome#388]
I'm going to just put everything I found into one issue.

Other programs
https://vimawesome.com/plugin/bash-language-server
https://vimawesome.com/plugin/cquery
https://vimawesome.com/plugin/ctags
https://vimawesome.com/plugin/eslint
https://vimawesome.com/plugin/jedi
https://vimawesome.com/plugin/moonscript
https://vimawesome.com/plugin/quickfix
https://vimawesome.com/plugin/racer
https://vimawesome.com/plugin/rls
https://vimawesome.com/plugin/rtags
https://vimawesome.com/plugin/rust
https://vimawesome.com/plugin/rustfmt
https://vimawesome.com/plugin/vscode-css-languageserver-bin

Emacs extensions
https://vimawesome.com/plugin/haskell-mode
https://vimawesome.com/plugin/spacemacs
https://vimawesome.com/plugin/cider
https://vimawesome.com/plugin/multiple-cursors-el
https://vimawesome.com/plugin/slime
https://vimawesome.com/plugin/bundler-el
https://vimawesome.com/plugin/ruby-refactor
https://vimawesome.com/plugin/emacs-theme-darktooth
https://vimawesome.com/plugin/python-el
https://vimawesome.com/plugin/powerline
https://vimawesome.com/plugin/clojure-mode
https://vimawesome.com/plugin/emacs-for-python
https://vimawesome.com/plugin/emacs-color-theme-solarized
https://vimawesome.com/plugin/helm - already mentioned in #396 but I still wanted to mention it here.

VSCode extensions
https://vimawesome.com/plugin/puppet-vscode
https://vimawesome.com/plugin/vscode-ccls
https://vimawesome.com/plugin/vscode-javascript
https://vimawesome.com/plugin/vscode-ocaml

bash/fish/zsh plugins
https://vimawesome.com/plugin/alias-tips
https://vimawesome.com/plugin/android-completion
https://vimawesome.com/plugin/antigen
https://vimawesome.com/plugin/auto-fu-zsh
https://vimawesome.com/plugin/bashmarks
https://vimawesome.com/plugin/bash-powerline
https://vimawesome.com/plugin/docker-zsh-completion
https://vimawesome.com/plugin/fast-syntax-highlighting
https://vimawesome.com/plugin/fish-getopts
https://vimawesome.com/plugin/fishnet
https://vimawesome.com/plugin/fzf-marks
https://vimawesome.com/plugin/get
https://vimawesome.com/plugin/git-flow-completion
https://vimawesome.com/plugin/lesaint-mvn
https://vimawesome.com/plugin/liquidprompt
https://vimawesome.com/plugin/oh-my-git
https://vimawesome.com/plugin/oh-my-zilsh
https://vimawesome.com/plugin/oh-my-zilsh-today-cannot-touch
https://vimawesome.com/plugin/oh-my-zsh
https://vimawesome.com/plugin/opp-zsh
https://vimawesome.com/plugin/powerlevel10k
https://vimawesome.com/plugin/powerlevel9k
https://vimawesome.com/plugin/powerline-bash
https://vimawesome.com/plugin/pure
https://vimawesome.com/plugin/segment
https://vimawesome.com/plugin/spot
https://vimawesome.com/plugin/statusline
https://vimawesome.com/plugin/virtualfish
https://vimawesome.com/plugin/z-fish
https://vimawesome.com/plugin/zsh-autopair
https://vimawesome.com/plugin/zsh-bundle-exec
https://vimawesome.com/plugin/zsh-completions
https://vimawesome.com/plugin/zsh-context-sensitive-alias
https://vimawesome.com/plugin/zsh-git-prompt
https://vimawesome.com/plugin/zsh-history-substring-search
https://vimawesome.com/plugin/zsh-kubernetes
https://vimawesome.com/plugin/zsh-syntax-highlighting

Not software
https://vimawesome.com/plugin/referencecard
https://vimawesome.com/plugin/fonts - powerline fonts. Just fonts.

**Comment by MaienM** [comment]
I ran into one of these just now, and was very confused by it for a bit. I wanted to report this problem, and was surprised to find that there is no built-in way to do so. I can submit a new entry, but I cannot remark on existing entries. 

I found #170, which was closed in favor of a different solution for the specific case raised there, but in general it might be useful to be able to report a (perceived) issue with an entry. Alternatively, checking whether a repository actually contains any vim files would be a reasonable way to automate this.

**Comment by Deathnerd** [comment]
Also https://github.com/PowerShell/PowerShell when searching for "PowerShell" on the website. That's an _entire language runtime_ . After a 10 minute investigation, I think it's the `^[^\S\n]*(?:Bundle|Plugin)[^\S\n]*[\'"]([^\'"\n\r]+)[\'"]` regex in the scraper that's generating this false positive

**Comment by nicolas-mace** [comment]
Yes, a lot of things that aren't plugins, and over a thousand pages pointing to repositories that no longer exist.
